.travis.yml

```diff
@@ -1,72 +0,0 @@
-language: python
```

Comment:
It is useful to keep this file for now, but we can delete the `before_deploy` and `deploy` sections. This runs tests on the CI services for new PRs.

Ideally, we could port these tests over to a GitHub Actions workflow named something like `.github/workflows/tests.yml`. About a year or so ago, TravisCI started more strictly limiting the amount of free minutes for open source projects, but given the low activity on this repository, it may still work okay for us.

Reply:
requires also keeping `travis/partial.sh`

Reply:
N/A after GHA